Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/deltaonboarding/ui/DeltaOnboardingFragmentViewModel.kt

```diff
@@ -59,6 +61,12 @@ class DeltaOnboardingFragmentViewModel @AssistedInject constructor(
         }
     }
 
+    fun isVaccinationRegistrationOnboardingDone() = vaccinationSettings.registrationAcknowledged
+
+    fun setVaccinationRegistrationOnboardingDone(value: Boolean) {
+        vaccinationSettings.registrationAcknowledged = value
+    }
+
```

Comment:
Can be simplified var isVaccinationRegistrationOnboardingDone = vaccinationSettings.registrationAcknowledged

Reply:
true, but I let it like this to have consistend look when we have more is_()/set_() methods in the `DeltaOnboardingFragmentViewModel`